modules/hal_hpmicro/Kconfig

```diff
@@ -0,0 +1,86 @@
+# Copyright (c) 2022 HPMicro
+# SPDX-License-Identifier: Apache-2.0
+config ZEPHYR_HAL_HPMICRO_MODULE
+	bool
+
+config HAS_HPMICRO_HAL
+	bool "HPMSDK HAL drivers support"
+
+if HAS_HPMICRO_HAL
+
+config USE_HPMSDK_UART
+	bool "HPMSDK UART driver support"
```

Comment:
these options should be selected by drivers, not directly. so prompt should be removed.

Reply:
Currently it is the selection by driver. For example. `GD` `Infineon` are written in this way.